_drafts/SC1-2-2-audio-captions.md

```diff
@@ -0,0 +1,77 @@
+---
+name: audio elements captions
+description: |
+  Non-streaming `audio` elements must not contain auditory information that is not expressed in captions or a text alternative.
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `audio` element [audible on the page][].
+
+### Expectation
+
+The auditory information of each test target is available through a caption, or through a [text transcript][] that is available either on the page or through a link.
+
```

Comment:
I would suggest splitting this out into separate atomic rules in a composed rule, where each atomic rule test for a separate method of living up to this success criterion, e.g. one for text transcript, one for captions made in one specific way, possibly another one for captions made in another way. Right now, this rule is as broad as the success criterion it is testing. 

Like for https://github.com/auto-wcag/auto-wcag/pull/158/files, I would also suggest splitting it out into separate expectations (or even separate rules), to test first that captions/transcript is available and then that it is actually descriptive.

Reply:
We started with two and actually ended up combining them because they are very similar. I see where you're coming from, but in my opinion it's not worth the overhead.